web/docs/Frontend Starters/Next.js/Next.js + Drupal/troubleshooting.md

```diff
@@ -51,3 +52,149 @@ Local Development:
 
     See
     [The docs on the `next/image` component for more information](https://nextjs.org/docs/api-reference/next/image#src).
+    ````
+
+## Adapting for Use With Existing Drupal Sites
+
+Our starter kits assume that you are using Drupal's core Media module to manage
+images for article content. If you are instead using Drupal's default image
+field, you will need to make the following adjustments to the starter kit:
+
+1. Update grid pages to use the `field_image` field instead of the
```

Comment:
I wonder if something like this will be common enough to write a codemod script for these changes?

Reply:
Had the same thought as well. Would be a nice future enhancement and would allow us to wipe out most of this section.